test/e2e/framework.go

```diff
@@ -1189,6 +1191,12 @@ func (b *PodBuilder) WithMutateFunc(f func(*corev1.Pod)) *PodBuilder {
 	b.MutateFunc = f
 	return b
 }
+func (b *PodBuilder) MountConfigMap(volumeMount []corev1.VolumeMount, volume []corev1.Volume) *PodBuilder {
```

Comment:
there is nothing specific to a ConfigMap here, so the name doesn't make sense.

the method should be replaces with 2 methods: `AddVolume(volume corev1.Volume)` and `AddVolumeMount(volumeMount corev1.VolumeMount)`.

**in addition to these**, if it is convenient for you you can always add a 3rd method:
```go
func (b *PodBuilder) MountConfigMap(configMapName string, mountPath string) *PodBuilder {
	volume := ...
	volumeMount := ...
	return b.AddVolume(volume).AddVolumeMount(volumeMount)
}
```

hopefully that all makes sense